lib/recurly.js

```diff
@@ -242,9 +236,40 @@ export class Recurly extends Emitter {
     }
   }
 
+  /**
+   * Disables a recurly.js instance
+   *
+   * - Removes all listeners
+   * - Propagates a destroy call to the bus
+   * - Stops the bus
+   * - Removes external references
+   *
+   * TODO:
+   *  - mark as destroyed, prevent subsequent destruction?
+   *  - destroy pricing instances, or let them live on? They
+   *    will not receive messages since the bus has been destroyed
+   */
+  destroy () {
+    debug('destroying Recurly instance', this.id);
+    this.off();
+    if (this.bus) {
+      this.bus.send('destroy');
+      this.bus.destroy();
+    }
+    if (this.fraud) {
+      this.fraud.destroy();
+    }
+    if (this.reporter) {
+      this.reporter.destroy();
+      delete this.reporter;
+    }
+  }
```

Comment:
Currently this will unbind listeners, send the destroy message along the bus, and instruct dependent `Fraud` and `Reporter` singleton associates to also destroy their listeners. For now, `recurly.Pricing` instances remain usable even after calling `recurly.destroy()`.

Reply:
The intent of this method is to make it available to a react component to easily unbind relevant listeners.